lib/jasmine-runner.js

```diff
@@ -1,6 +1,16 @@
 /* globals jasmine, phantom */
+
+var args;
+
+if (typeof phantom.args === 'undefined') {
+    //Compatible with phantomjs ^2.0
```

Comment:
Change the comment to 'compatible with phantom version at least 2.0' And comment this in the code: 'free && sync && echo 3 > /proc/sys/vm/drop_caches && free' to release mem in case crash again. Tried on remote slave phantomjs20 works fine. But I'm gonna do https://c3energy.atlassian.net/browse/TOOLS-2068, so we may directly use higher version in this or next week on all slaves